.github/workflows/iroha2-pr-ui.yml

```diff
@@ -15,10 +15,29 @@ concurrency:
 
 env:
   CARGO_TERM_COLOR: always
+  DOCKER_COMPOSE_PATH: defaults
 
 jobs:
+  build_executor:
```

Comment:
I don't think executor is required to execute `ui` tests

Reply:
What is the resolution here? I see it was agreed that the executor is not needed, but the job is still here

Reply:
It was removed [here](https://github.com/hyperledger/iroha/pull/4931/commits/c1490f414a4b103676d56109a8254476b24ee4e9#diff-f4db2cb3367415c444df62a5359b0ca64b4930f0dfb52e29c652ebc921804722) and reverted [here](https://github.com/hyperledger/iroha/pull/4931/commits/bca9adbd20620e4cb4fb9bd96981e57620c87dc8)

Reply:
So we do need the executor for UI tests?

Reply:
Removed